components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,86 @@
+@import "@not-govuk/sass-base";
+@import "govuk-frontend/govuk/helpers/_colour.scss";
```

Comment:
I don't think you should need this line.

What do you think you need it?

Reply:
Shouldn't this provide the values for some of your _"magic constants!"_ later on?

Reply:
Probably, but those constants are already imported via the sass-base.

Reply:
ThIs was to import the govuk colours. Have removed it now as line 1 `@import "@hods/sass-base";` also does this.